### PR TITLE
OpenLLM

### PR DESCRIPTION
**Pull request by rxm7706**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rxm7706** [comment]
**OSX -** 
```
Encountered problems while solving:
  - nothing provides requested bitsandbytes <0.42
  - nothing provides requested cuda-python
```
**WIN**
```
Encountered problems while solving:
  - nothing provides requested bitsandbytes <0.42
  - nothing provides pytorch >=1.9 needed by optimum-1.13.2-pyhd8ed1ab_0
  - nothing provides pytorch >=1.4.0 needed by accelerate-0.10.0-pyhd8ed1ab_0
```

File: recipes/openllm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    - einops
+    - sentencepiece
+    - scipy
+    - python-build <1
+    - click >=8.1.3
+    - cuda-python
+    - bitsandbytes <0.42
+# Required for openllm-playground --help
+# https://github.com/bentoml/OpenLLM/blob/main/openllm-python/pyproject.toml#L119
+    - jupyter
```
Comment (xhochy) [inline recipes/openllm/meta.yaml:52]: This should be a different output then. A project that is meant for production shouldn't have debug dependencies.

**Review by xhochy — CHANGES_REQUESTED** [review]
`core` and `client` are fine. If the discussion is not quickly resolved, submit them as separate PRs to get them merged faster.

File: recipes/openllm/meta.yaml (hunk 1 above)
Comment (rxm7706) [reply recipes/openllm/meta.yaml]: @xhochy - 
Makes sense - given that playground had a client entry point and help command as part of the main package - I wasn't sure how to treat it, as a feature or a debug / separate from production. 
    - openllm-playground = openllm_cli.extension.playground:cli
All the same - let me rework this PR

Thanks for your time, I will revert back.

**Comment by rxm7706** [comment]
@xhochy - Updated to split out openllm-playground - Please review!
Thanks and Regards.

File: recipes/openllm-playground/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
+    - hatchling ==1.18.0
+    - hatch-vcs ==0.3.0
+    - hatch-fancy-pypi-readme ==23.1.0
+    - pip
```
Comment (xhochy) [inline recipes/openllm-playground/meta.yaml:32]: Otherwise they will clobber eachother.
```suggestion
    - pip
    - openllm {{ version }}
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Review by xhochy — APPROVED** [review]

**Comment by rxm7706** [comment]
@conda-forge/help-python, ready for review!

**Pull request merged**